src/community/china/_os-settings.md

```diff
@@ -0,0 +1,124 @@
+   {% assign prompt = '$' -%}
+   {% assign comtoset = 'export ' -%}
+   {% assign installdirsuggestion = '`~/dev`' %}
+   {% capture envvarset -%}{{prompt}} {{comtoset}}{% endcapture -%}
+   {% capture setpath -%}{{envvarset}}PATH="$PWD/flutter/bin:$PATH"{% endcapture -%}
+   {% capture newdir -%}{{prompt}} mkdir ~/dev{% endcapture -%}
+   {% if id == 'macos' %}
+      {% capture unzip -%} {{prompt}} unzip{% endcapture -%}
+   {% else %}
+      {% capture unzip -%} {{prompt}} tar -xf{% endcapture -%}
+   {% endif %}
+   {% capture permaddexample -%}
+cat <<EOT >> ~/.zprofile
+{{envvarset}}PUB_HOSTED_URL="https://pub.flutter-io.cn"
+{{envvarset}}FLUTTER_STORAGE_BASE_URL="https://storage.flutter-io.cn"
+{{setpath}}
+EOT
+   {% endcapture -%}
+{% endif -%}
+{%- case id %}
+   {% when 'windows','macos' %}
+      {%- assign file-format = 'zip' %}
+      {%- assign download-os = id %}
+   {% when 'linux','chromeos' %}
+      {%- assign download-os = 'linux' %}
+      {%- assign file-format = 'tar.xz' %}
+{% endcase %}
+
+<div id="{{id}}" class="tab-pane
+  {%- if id == 'windows' %} active {% endif %}"
+  role="tabpanel" aria-labelledby="{{id}}-tab" markdown="1">
+
+This procedure requires using {{shell}}.
+
+1. Open a new window in {{shell}} to prepare running scripts.
+
+1. Set `PUB_HOSTED_URL` to your mirror site.
+
+   ```terminal
+   {{envvarset}}PUB_HOSTED_URL="https://pub.flutter-io.cn"
+   ```
+
+1. Set `FLUTTER_STORAGE_BASE_URL` to your mirror site.
+
+   ```terminal
+   {{envvarset}}FLUTTER_STORAGE_BASE_URL="https://storage.flutter-io.cn"
+   ```
+
+1. Download the Flutter archive from your mirror site.
+   In your preferred browser, go to
+   [Flutter SDK archive](https://flutter.cn/docs/release/archive?tab={{id}}).
+
+1. Create a folder where you can install Flutter. then change into it.
+
+   Consider a path like {{installdirsuggestion}}.
+
+   ```terminal
+   {{newdir}}; cd {{installdirsuggestion}}
+   ```
+
+1. Extract the the SDK from the {{file-format}} archive file.
+
+   This example assumes you downloaded the {{os}} version of the Flutter SDK.
+
+   ```terminal
+   {{unzip}} {{sdk | replace: "opsys", download-os}}{{file-format}}
+   ```
+
+1. Add Flutter to your `PATH` environment variable.
+
+   ```terminal
+   {{setpath}}
+   ```
+
+1. Run Flutter Doctor to verify your installation.
+
+   ```terminal
+   {{prompt}} flutter doctor
+   ```
+
+1. Return to the [setting up Flutter]({{site.url}}/get-started/editor)
+   guide and continue from that procedure.
+
+From this example, `flutter pub get` fetches packages from `flutter-io.cn`,
+in any terminal where you set `PUB_HOSTED_URL` and `FLUTTER_STORAGE_BASE_URL`.
+
+Any environment variables set using `{{comtoset}}` in this procedure
+only apply to the current window.
+
+To set these values on a permanent basis,
+{% if id == 'windows' -%}
+set the enviroment variables as in the following example:
+{% else -%}
+append those three `export` commands to the `*rc` or `*profile`
```

Comment:
It's been a while since the `export` commands at this point. I wasn't sure what you were referring to at first.

```suggestion
append the previous three `export` commands to the `*rc` or `*profile`
```